iceoryx_hoofs/include/iceoryx_hoofs/internal/log/logstream.inl

```diff
@@ -200,6 +200,27 @@ inline LogStream& LogStream::operator<<(const LogLevel value) noexcept
     return *this;
 }
 
+namespace internal
```

Comment:
I thought you like

```suggestion
namespace detail
```

more :grin:

Reply:
Hahaha. Good point but only after it is moved to the new location :)

If another commit is required I'll do it with this PR.

Reply:
Hey, it is me the party pooper :smile: 

I my opinion we gain little to nothing with introducing `detail` as internal namespace. 
But we have to pay for it with an non insignificant amount of work.

Don't get me wrong here, I am not against it but could we start with this when we are once bored again and are looking for something to do?